package.json

```diff
@@ -55,6 +56,7 @@
     "retry": "0.6.1",
     "strophe": "1.2.4",
     "strophejs-plugins": "0.0.7",
+    "styled-components": "1.3.0",
```

Comment:
Hum, probably a good idea to bring up the issue in `@atlaskit/util-shared-styles` ?